sqlalchemy_exasol/merge.py

```diff
@@ -1,3 +1,8 @@
+# Refactoring notes
```

Comment:
Are these still needed?

Reply:
I would mostly ignore this file for now, with the changed version it needs a complete rework. I can roll it back to its original state though until then. What do you think?

Reply:
ah ok, in that case, I think you can keep the refactoring, because it is way better to understand than the original and might make it easier to rewrite it.